src/OpenTracing/IScope.cs

```diff
@@ -0,0 +1,95 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AbstractTag.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing
+{
+    using System;
+
+    /// <summary>
+    ///     A <see cref="IScope" /> formalizes the activation and deactivation of a <see cref="ISpan" />, usually from a CPU
+    ///     standpoint.
+    ///     <para>
+    ///         Many times a <see cref="ISpan" /> will be extant (in that <see cref="ISpan.Finish()" /> has not been called)
+    ///         despite being in a
+    ///         non-runnable state from a CPU/scheduler standpoint. For instance, a <see cref="ISpan" /> representing the child
+    ///         side of an
+    ///         RPC will be unfinished but blocked on IO while the RPC is still outstanding. A <see cref="IScope" /> defines
+    ///         when a given
+    ///         <see cref="ISpan" /> <em>is</em> scheduled and on the path.
+    ///     </para>
+    ///     <para>
+    ///         Calling <see cref="IScope.Dispose" /> marks the end of the active period for the current thread and
+    ///         <see cref="IScope" />,
+    ///         updating the <see cref="IScopeManager.Active" /> in the process.
+    ///     </para>
+    /// </summary>
+    public interface IScope : IDisposable
+    {
+        /// <summary>
+        ///     The <see cref="ISpan" /> that's been scoped by this <see cref="IScope" />
```

Comment:
Are you using ReSharper or some other custom style conventions? Visual Studio does not intend comments and it doesn't add spaces before a "/>" by default. I'd prefer using the defaults since that's what Microsoft (.NET Core) and most people use.

Reply:
Baaaaah I hate the standards that use "XML"doc and don't use XML indentations, so hard to read! But that's fine everyone seems to agree with you so I'll make the tags line up to the text (sad face)

Reply:
I know... I had my own (different) preferences (tabs, anyone?!? :smile:) for a long time but then I just gave up and now I'm just trying to do whatever the VS defaults do as that is what most people have. :|

Reply:
resolved